tests/std/tests/P2467R1_exclusive_mode_fstreams/test.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <filesystem>
+#include <fstream>
+
+namespace fs = std::filesystem;
+
+#define FILENAME L"test.dat"

```

Comment:
In modern C++ style, I strongly prefer to use `string_view`s to macros for string constants. That said, this is only ever used to initialize `path`s, so this could as well be a `path` constant, simplifying the rest of the code a bit.